packages/embed/src/utils/set-auto-close.ts

```diff
@@ -0,0 +1,10 @@
+import { getFormSubmitHandler } from './create-iframe/get-form-event-handler'
+
+export const setAutoClose = (embedId: string, options: { autoClose?: number }, callback: () => void) => {
```

Comment:
Lets not pass all options, just the necessary one. Also lets rename the callback since it is always `close` function:

```suggestion
export const setAutoClose = (embedId: string, autoClose: number, closeCallback: () => void) => {
```